mne/proj.py

```diff
@@ -97,15 +97,21 @@ def _compute_proj(data, info, n_grad, n_mag, n_eeg, desc_prefix, verbose=None):
         if n == 0:
             continue
         data_ind = data[ind][:, ind]
-        U = linalg.svd(data_ind, full_matrices=False,
-                       overwrite_a=True)[0][:, :n]
-        for k, u in enumerate(U.T):
+        # data is the covariance matrix: U * S**2 * Ut
+        U, Sexp2, _ = linalg.svd(data_ind, full_matrices=False,
+                                 overwrite_a=True)
+        U = U[:, :n]
+        exp_var_ = Sexp2 / Sexp2.sum()
+        exp_var_ = exp_var_[:n]
+        for k, temp in enumerate(zip(U.T, exp_var_)):
+            u, var = temp
             proj_data = dict(col_names=names, row_names=None,
                              data=u[np.newaxis, :], nrow=1, ncol=u.size)
             this_desc = "%s-%s-PCA-%02d" % (desc, desc_prefix, k + 1)
             logger.info("Adding projection: %s" % this_desc)
             proj = Projection(active=False, data=proj_data,
                               desc=this_desc, kind=1)
+            proj.explained_variance_ = var
```

Comment:
Why put the trailing `_` on the variable name? I know `sklearn` does it, but we don't typically anywhere other than `sklearn`-style classes AFAIK.